src/index.html

```diff
@@ -11,7 +11,192 @@
+        <div class="form__block">
+          <label for="email" class="form__label">
+              E-mail:
+          </label>
+          <input
+            type="email"
+            id="email"
+            autocomplete="off"
+            placeholder="email@example.com"
+          >
+        </div>
+        <div class="form__block">
+          <label for="password" class="form__label">
+            Password:
+          </label>
+          <input
+            type="password"
+            id="password"
+            required
+          >
+        </div>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>An interesting fact about you!</legend>
+        <div class="form__block">
+          <label class="form__label">
+            Do you love cats?
+          </label>
+          <label for="yes">
+            Yes
+            <input
+              type="radio"
+              id="yes"
+              name="radio"
+              required
+              value="yes"
+            >
+          </label>
+          <label for="no">
+            No
+            <input
+              type="radio"
+              id="no"
+              name="radio"
+              required
+              value="no"
+            >
+          </label>
+        </div>
+        <div class="form__block">
+          <label for="color" class="form__label">
+            What is your favorite color?
+          </label>
+          <input
+            type="color"
+            id="color"
+            required
+          >
+        </div>
+        <div class="form__block">
+          <label for="sleep" class="form__label">
+            What time do you go to bed?
+          </label>
+          <input
+            type="time"
+            id="sleep"
+            required
+          >
+        </div>
+        <div class="form__block">
+          <label class="form__label">
+            What are your favorite brand of cars?
+          </label>
+          <select
+            name="cars"
+            multiple
+            required
+          >
+            <option value="bmw">BMW</option>
+            <option value="audi">Audi</option>
+            <option value="lada">Lada</option>
+            <option value="empty"></option>
+          </select>
+        </div>
+        <div class="form__block">
+          <label for="rating" class="form__label">
+            How do you rate our work?
+          </label>
+          <input
+            type="range"
+            id="rating"
+            value="0"
+            required
+          >
+        </div>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>Additional info:</legend>
+        <div class="form__block">
+          <label class="form__label">
+            Comments:
+          </label>
+          <textarea
+            name="comment"
+            cols="20"
+            rows="2"
+          >
+          </textarea>
+        </div>
+        <div class="form__block">
+          <label class="form__label">
```

Comment:
`#div-wrappers` (use this reference to see related comments)
you're over-bloating your code. There are 14 div wrappers in total - open + close tags make it up 28 lines of code that you could avoid.
You're not using the label class for styling.
So the easy-fix is to
- get rid of all `<div` wrappers
- and style the class `form__label` - just add a `display` rule to display all labels as blocks
```suggestion
        <label class="form__label">
```